crates/bevy_app/src/plugin.rs

```diff
@@ -6,14 +6,22 @@ use std::any::Any;
 /// A collection of Bevy app logic and configuration.
 ///
 /// Plugins configure an [`App`]. When an [`App`] registers a plugin,
-/// the plugin's [`Plugin::build`] function is run.
+/// the plugin's [`Plugin::build`] function is run. By default, a plugin
+/// can only be added once to an [`App`]. If the plugin may need to be
+/// added twice or more, the function [`is_unique`](Plugin::is_unique)
+/// should be overriden to return `false`.
```

Comment:
```suggestion
/// should be overriden to return `false`.
///
/// Each copy of a generic plugins has a different type (if and only if their stored types are different),
/// and so will not be considered duplicated unless multiple copies with the same generic are added.
```

Reply:
it's a tiny bit more complicated than that, I tried pushing more docs. are they ok for you?

Reply:
Yep that's fine.